_health-care/_js/components/form-elements/ErrorableNumberInput.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import _ from 'lodash';
+
+/**
+ * A form input with a label that can display error messages.
+ *
+ * Props:
+ * `errorMessage` - Error string to display in the component.
+ *                  When defined, indicates input has a validation error.
+ * `label` - String for the input field label.
+ * `max` - String or function.
+ * `min` - String or function.
+ * `pattern` - String specifying the pattern for the input.
+ * `placeholder` - placeholder string for input field.
+ * `required` - boolean. Render marker indicating field is required.
+ * `value` - string. Value of the input field.
+ * `onValueChange` - a function with this prototype: (newValue)
+ */
+class ErrorableNumberInput extends React.Component {
+  constructor() {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.inputId = _.uniqueId('errorable-number-input-');
+  }
+
+  handleChange(domEvent) {
+    this.props.onValueChange(domEvent.target.value);
+  }
+
+  render() {
+    // Calculate error state.
+    let errorSpan = '';
+    let errorSpanId = undefined;
+    if (this.props.errorMessage) {
+      errorSpanId = `${this.inputId}-error-message`;
+      errorSpan = <span className="usa-input-error-message" id={`${errorSpanId}`}>{this.props.errorMessage}</span>;
+    }
+
+    // Calculate required.
+    let requiredSpan = '';
+    if (this.props.required) {
+      requiredSpan = <span className="usa-additional_text">Required</span>;
+    }
+
+    return (
+      <div className={this.props.errorMessage ? 'usa-input-error' : undefined}>
+        <label
+            className={this.props.errorMessage !== undefined ? 'usa-input-error-label' : undefined}
```

Comment:
Same as another comment, "Will this make the field look wrong until the user types something in because it will be undefined on initialization correct?"